### PR TITLE
Fix bugs Stocks Candle moving average

### PR DESCRIPTION
**Pull request by minhhoang1023**
# Description

Problems described in [issue](https://github.com/GamestonkTerminal/GamestonkTerminal/issues/1362)

Solutions:
- Add try except clause to present code from breaking
- Add Legend & Title to the plot
- Update description of ma arguments for clarity

image

- [x] Summary of the change / bug fix.
- [x] Link # issue, if applicable.
- [x] Screenshot of the feature or the bug before/after fix, if applicable.
- [x] Relevant motivation and context. 
- [x] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes. 
* Provide instructions so we can reproduce. 
* Please also list any relevant details for your test configuration.

# Checklist:

- [x] Update [our Hugo documentation](https://gamestonkterminal.github.io/GamestonkTerminal/) following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [ ] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

**Comment by jmaslek** [comment]
2 Comments here:

1) Can we label the legend something like f"MA {ma}" instead of just the number?
2) Adding the trendlines should not add the second axes to the left as shown in the screenshot.

**Review by DidierRLopes — APPROVED** [review]

**Review by piiq — APPROVED** [review]

**Comment by minhhoang1023** [comment]
Secondary y axis is removed & legend is more clear.

image

File: gamestonk_terminal/stocks/stocks_controller.py [hunk 1]
```diff
@@ -231,8 +231,8 @@ def call_candle(self, other_args: List[str]):
             "--ma",
             dest="mov_avg",
             type=str,
-            help="Add moving averaged to plot",
-            default="",
+            help="Add moving average in number of days to plot and separate by a coma. Example: 20,30,50",
```
Comment (jmaslek) [inline gamestonk_terminal/stocks/stocks_controller.py:234]: comma*

Comment (minhhoang1023) [reply gamestonk_terminal/stocks/stocks_controller.py]: lol coma 😴

**Pull request merged**